core/frontend/src/components/video-manager/VideoThumbnail.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <v-container class="d-flex flex-column align-center justify-space-between">
+    <v-avatar
+      :height="height"
+      :width="width"
+      tile
+    >
+      <img
+        v-if="thumbnail?.status === 200"
+        :src="thumbnail?.source"
+      >
+      <span
+        v-else-if="thumbnail?.status === 404 || (thumbnail?.source === undefined && thumbnail?.status !== 503)"
```

Comment:
It would be nice to have the ifs here as a properties for better readability of the ui description.

Reply:
done